### PR TITLE
Server: Special mode with empty listen_host : try listen v4 and v6 

### PR DESCRIPTION
**Pull request by proller**
without fatal error if protocol not supported (for docker and freebsd)

### CODE REVIEW

**Comment by proller** [comment]
will auto-reformat after merge

File: dbms/src/Server/Server.cpp [hunk 1]
```diff
@@ -547,8 +549,18 @@ int Server::main(const std::vector<std::string> & args)
 
                 LOG_INFO(log, "Listening interserver: " + interserver_address.toString());
             }
+
+            } catch (const std::exception & e) {
```
Comment (alexey-milovidov) [inline dbms/src/Server/Server.cpp]: Need to catch only relevant exceptions.

**Review by alexey-milovidov — CHANGES_REQUESTED** [review]
.

**Comment by alexey-milovidov** [comment]
You need to check exception code.
Now the behaviour is wrong for case of "Address already in use" error: half of server interfaces could be started.

**Pull request merged**